src/quo/components/wallet/amount_input/schema.cljs

```diff
@@ -0,0 +1,17 @@
+(ns quo.components.wallet.amount-input.schema)
+
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map {:closed true}
+      [:status {:optional true} [:enum :default :error]]
```

Comment:
usually if a prop is optional, we tend to add :maybe to it to assure that nil will also pass the schema check. This would become:
```clj
[:status {:optional true} [:maybe [:enum :default :error]]]

```